src/material/chips/chip.scss

```diff
@@ -695,16 +695,20 @@ $_avatar-trailing-padding: 8px;
 
   // Used as a state overlay.
   &::after {
-    $offset: 2px;
+    $_touch-target-size: 48px;
+    $_ripple-size: 24px;
     content: '';
     display: block;
     opacity: 0;
     position: absolute;
-    top: 0 - $offset;
-    bottom: 0 - $offset;
-    left: 8px - $offset;
-    right: 8px - $offset;
-    border-radius: 50%;
+    // Increases touch target size to improve accessibility and fix b/286959517
+    z-index: 100;
```

Comment:
I don't think `z-index` or `pointer-events` is actually needed, I just put that in my original snippet to be extra safe

Reply:
Removed